okhttp-android-support/pom.xml

```diff
@@ -17,6 +17,12 @@
 
   <dependencies>
     <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+      <version>3.0.2</version>
```

Comment:
You don't need to repeat the version in all these though, right? The dependency management in the root will take care of that